db/schema.rb

```diff
@@ -11,20 +11,18 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20150204075501) do
+ActiveRecord::Schema.define(version: 20150306060237) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
   create_table "instances", force: true do |t|
-    t.string "name", null: false
     t.string "host", null: false, comment: "Stores the host name of the instance. The www prefix is automatically handled by the application"
+    t.string "name", null: false
```

Comment:
You have changed too many columns which are not supposed to be changed, please git diff first before commit.

I guess this was caused by that you are using <code>rake db:migrate </code> instead of <code> rake db:schema:load </code> ?